### PR TITLE
update for tul_cob & log shipping

### PR DESCRIPTION
**Pull request by cmharlow**
Work @sensei100 & myself did to be able to use this role with tul_cob, then to add AWS Cloudwatch log shipping. Blocking tulibraries/tul_cob_playbook#148

### CODE REVIEW

**Comment by cmharlow** [comment]
Removed log shipping from here to use an existing role at playbook level.

File: handlers/main.yml [hunk 1]
```diff
@@ -9,6 +23,7 @@
     - assets:clobber
     - assets:precompile
     - db:migrate
+    - db:seed
```
Comment (bibliotechy) [inline handlers/main.yml]: I actually think `db:seed` is dangerous, and we should not do it in production.

Comment (cmharlow) [reply handlers/main.yml]: it was running in production on the current tul_cob playbook.

Comment (cmharlow) [reply handlers/main.yml]: removed.

**Comment by cmharlow** [comment]
@bibliotechy back to you

**Pull request merged**